server/index.js

```diff
@@ -188,6 +188,39 @@ app.get(`/quiz/history/taken/:userId`, async (req, res) => {
   }
 })
 
+app.get(`/quiz/rankings/global/:quizId`, async (req, res) => {
+  try {
+    const quizId = req.params.quizId;
+    const retrieveQuizGlobalRankings = await pool.query(
+      `SELECT * FROM user_completed_quizzes
+      WHERE id_quiz = ${quizId}`
+    )
+    res.status(200).send(retrieveQuizGlobalRankings);
```

Comment:
If you're sending a 200 status, you can just res.send, which will automatically send a 200.